lib/OpenQA/WebAPI.pm

```diff
@@ -229,7 +229,7 @@ sub startup {
 
     unshift @{$self->renderer->paths}, '/etc/openqa/templates';
 
-    $self->plugin('AssetPack');
+    $self->plugin(AssetPack => {pipes => [qw(Sass Css JavaScript Combine)]});
```

Comment:
Not sure how to document this, but pretty sure it works better if you do (JavaScript Css Sass). You should not process a "css" file produced by the "Sass" pipe.